site/content/docs/5.3/content/tables.md

```diff
@@ -521,7 +521,7 @@ Responsive tables make use of `overflow-y: hidden`, which clips off any content
 Across every breakpoint, use `.table-responsive` for horizontally scrolling tables.
 
 <div class="bd-example">
-  <div class="table-responsive">
+  <div class="table-responsive" tabindex="0">
```

Comment:
Not sure about adding `tabindex="0"` everywhere.
It seems not yet ok on our side (https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1711) and on Boostrap's side (https://github.com/twbs/bootstrap/pull/38971).

Reply:
Ok, I'll remove this rule and hide some more elements.

Reply:
I thinks it's a safer option for now.